parsec/api/protocol/organization.py

```diff
@@ -52,6 +53,7 @@ class OrganizationBootstrapWebhookSchema(BaseSchema):
     device_label = fields.String(allow_none=True, required=True)
     human_email = fields.String(allow_none=True, required=True)
     human_label = fields.String(allow_none=True, required=True)
+    users_limit = fields.Integer(allow_none=True, required=True)
```

Comment:
The expiration date is always set by the SaaS instead of the user's limit that is set by the backend. The SaaS need to know the limit to send its mails.